build.gradle

```diff
@@ -256,6 +259,31 @@ subprojects {
   }
 }
 
+// If "-P verboseTestOutput=true" is passed in, configure all subprojects to dump all of their
+// output and final test status (pass/fail, errors) for each test class.
+if (verboseTestOutput) {
+  gradle.projectsEvaluated({
+    subprojects {
```

Comment:
What's the reasoning for putting this here instead of in the subprojects block above?

Reply:
Yeah, I guess that merits a comment, which I've added.  The short answer is we have to apply these to all of the test targets, and they aren't defined when the main subprojects section is run.